openmdao/components/ks.py

```diff
@@ -0,0 +1,129 @@
+"""
+KS Function Component.
+"""
+
+import numpy as np
+
+from openmdao.core.explicitcomponent import ExplicitComponent
+
```

Comment:
I wonder if this would be a good place to use the CITATION feature, similar to it's use in `petsc_vector.py`

Reply:
Done